### PR TITLE
Add dirty code

### PR DESCRIPTION
**Pull request by pocke**

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def foo
```
Comment (wata727) [inline main.rb:1]: **[staging][rubocop]**
- Missing frozen string literal comment.

Comment (sumyapp) [inline main.rb:1]: **[rubocop]**
- Missing frozen string literal comment.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  eval(foo)
+end
```
Comment (wata727) [inline main.rb:3]: **[staging][rubocop]**
- The use of `eval` is a serious security risk.

Comment (sumyapp) [inline main.rb:3]: **[rubocop]**
- The use of `eval` is a serious security risk.

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+class AAA
+        def initialize
```
Comment (wata727) [inline main.rb:6]: **[staging][rubocop]**
- Missing top-level class documentation comment.
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

Comment (sumyapp) [inline main.rb:6]: **[rubocop]**
- Missing top-level class documentation comment.
- Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

File: main.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (sumyapp) [inline main.rb:7]: **[rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: main.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
```
Comment (sumyapp) [inline main.rb:8]: **[rubocop]**
- Extra empty line detected at method body beginning.

File: main.rb (hunk 4 above)
Comment (wata727) [inline main.rb:7]: **[staging][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: main.rb (hunk 5 above)
Comment (wata727) [inline main.rb:8]: **[staging][rubocop]**
- Extra empty line detected at method body beginning.

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/40/diff/0f24aca44618aa993a0eb72a2ac4c954c2f01426?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/40/diff/0f24aca44618aa993a0eb72a2ac4c954c2f01426?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

**Pull request closed without merging**